php/commands/scaffold.php

```diff
@@ -227,13 +232,19 @@ function _s( $args, $assoc_args ) {
 		$this->maybe_create_themes_dir();
 
 		$this->init_wp_filesystem();
-		unzip_file( $tmpfname, $theme_path );
+		$unzip_result = unzip_file( $tmpfname, $theme_path );
 		unlink( $tmpfname );
 
-		WP_CLI::success( "Created theme '{$data['theme_name']}'." );
+		if ( TRUE === $unzip_result ) {
+			WP_CLI::success( "Created theme '{$data['theme_name']}'." );
+		} else {
+			WP_CLI::error( "Could not decompress your theme files ('$tmpfname') at '$theme_path': " . $unzip_result->get_error_message() );
```

Comment:
If you're including variables in a string, can you wrap them in braces? `'{$tmpfname}'`